packages/create-sitecore-jss/src/templates/angular-xmcloud/angular.json

```diff
@@ -0,0 +1,34 @@
+{
+  "projects": {
+    "<%- appName %>": {
+      "architect": {
+        "build": {
+          "options": {
+            "styles": [
+              "node_modules/bootstrap/dist/css/bootstrap.min.css",
+              "src/styles.css",
+              "src/assets/styles/main.scss"
+            ],
+            "stylePreprocessorOptions": {
+              "includePaths": ["src/assets/styles", "node_modules"]
+            }
+          }
+        },
+        "test": {
```

Comment:
Do we need "stylePreprocessorOptions" defined for "test" as well?

Reply:
i think so, adding it.